apps/angular/src/app/features/auth/register/register.component.ts

```diff
@@ -0,0 +1,92 @@
+import { Component, DestroyRef, inject } from '@angular/core';
+import { takeUntilDestroyed } from '@angular/core/rxjs-interop';
+import { FormGroup, NonNullableFormBuilder, Validators } from '@angular/forms';
+import { Router } from '@angular/router';
+import { UserService } from '@js-camp/angular/core/services/user.service';
+import { AppValidators } from '@js-camp/angular/core/utils/app-validators';
+import { catchFormErrors } from '@js-camp/angular/core/utils/catch-form-error';
+import { ControlsOf } from '@js-camp/angular/core/utils/types/controls-of';
+import { AppValidationError } from '@js-camp/core/models/app-error';
+import { Register } from '@js-camp/core/models/auth/register';
+import { BehaviorSubject, catchError, finalize, first, throwError } from 'rxjs';
+
+import { MIN_PASSWORD_LENGTH } from '../utils/constants';
+
+type RegisterForm = ControlsOf<Register & { repeatPassword: string; }>;
+
+/** Register page. */
+@Component({
+	selector: 'camp-login',
+	templateUrl: './register.component.html',
+	styleUrls: ['../auth.css'],
+})
+export class RegisterComponent {
+	/** Loading status. */
+	protected readonly isLoading$ = new BehaviorSubject<boolean>(false);
+
+	/** Register form. */
+	protected readonly registerForm: FormGroup<RegisterForm>;
```

Comment:
_register_ sounds like a verb which is more common for functions
```suggestion
	protected readonly registrationForm: FormGroup<RegisterForm>;
```